src/App/FreeCADInit.py

```diff
@@ -33,10 +33,25 @@
 # imports the one and only
 import FreeCAD
 
+def removeFromPath(module_name):

```

Comment:
it's not used yet. Modules which want to use ```from FreeCAD import module``` can place the function in the Init.py. please read this thread: https://forum.freecadweb.org/viewtopic.php?f=10&t=21128